### PR TITLE
Rework reports descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request reworks reports API descriptions, and related files to conform to the [writing style](https://preview.redoc.ly/all-rebilly/rework-reports/tag/Reports).

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/CumulativeSubscriptions.yaml [hunk 1]
```diff
@@ -18,4 +19,7 @@ properties:
           description: Number of canceled subscriptions within the aggregation.
         cumulativeCount:
           type: integer
-          description: Number of cumulative subscriptions from the previous aggregation + number of new subscriptions - number of canceled subscriptions within the aggregation.
+          description: |-
+            Total number of cumulative subscriptions.
+
+            This field is calculated based on: the number of cumulative subscriptions from the previous aggregation period + `newCount` - `canceledCount`.
```
Comment (vykhrystiuk) [inline openapi/components/schemas/CumulativeSubscriptions.yaml:25]:
```suggestion
            This field is calculated based on the number of cumulative subscriptions from the previous aggregation period + `newCount` - `canceledCount`.
```

File: openapi/components/schemas/ReportMonthlyRecurringRevenue.yaml [hunk 2]
```diff
@@ -18,41 +19,58 @@ properties:
...
+            The difference between the current period MRR and the previous period MRR is equal to:
+            `new` + `reactivation` + `expansion` - `contraction` - `churned`.
           type: object
           properties:
             new:
               description: |-
-                MRR of new customers.
-                Example: A new customer signs up to a $40 per month plan. This is $40 in new MRR.
+                Occurs when new customers sign up.
+                New MMR is the amount of the new customer's subscription.
```
Comment (adamaltman) [inline openapi/components/schemas/ReportMonthlyRecurringRevenue.yaml:29]:
```suggestion
                New MRR is the amount of the new customer's subscription.
```

File: openapi/components/schemas/ReportMonthlyRecurringRevenue.yaml [hunk 3]
```diff
@@ -18,41 +19,58 @@ properties:
...
+                This is a new MRR amount of $40.
               type: number
               format: double
               example: 40.0
             reactivation:
               description: |-
-                Previous customer stops being a customer for a period.
-                The same customer signs up again and purchases a $25 per month plan. This is reactivation MRR of $25.
+                Occurs when a customer stops being a customer for a period but later signs up again.
+                Reactivation MMR is the amount of the customer's new subscription.
```
Comment (adamaltman) [inline openapi/components/schemas/ReportMonthlyRecurringRevenue.yaml:39]:
```suggestion
                Reactivation MRR is the amount of the customer's new subscription.
```

File: openapi/components/schemas/ReportMonthlyRecurringRevenue.yaml [hunk 4]
```diff
@@ -18,41 +19,58 @@ properties:
...
               type: number
               format: double
               example: 25.0
             churned:
               description: |-
-                Customer decides to cancel.
-                The subscription churns, the customer goes from $80 per month to $0 per month. This is $80 of churned MRR.
+                Occurs when a subscription is churned.
+                Churn occurs when the paid service period on a subscription order expires.
+                Churned MMR is the amount that is lost as a result of the churned subscription.
```
Comment (adamaltman) [inline openapi/components/schemas/ReportMonthlyRecurringRevenue.yaml:50]:
```suggestion
                Churned MRR is the amount that is lost as a result of the churned subscription.
```

**Comment by noon-dawg** [comment]
@Rebilly/php please review

File: openapi/components/schemas/ReportJournal.yaml [hunk 5]
```diff
@@ -13,14 +13,14 @@ properties:
     type: string
     description: |-
       Year and month when revenue is booked.
-      If empty then booked revenue starts from the first booked amount.
+      If this value is omitted, booked revenue starts is recorded from the first booked amount.
```
Comment (Boydbueno) [inline openapi/components/schemas/ReportJournal.yaml:16]:
```suggestion
      If this value is omitted, booked revenue is recorded from the first booked amount.
```

**Review by Boydbueno — APPROVED** [review]

**Comment by noon-dawg** [comment]
@Rebilly/php please review

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**